src/dotnet/commands/dotnet-run/LaunchSettings/LaunchSettingsManager.cs

```diff
@@ -25,7 +25,13 @@ public static LaunchSettingsApplyResult TryApplyLaunchSettings(string launchSett
         {
             try
             {
-                using (var document = JsonDocument.Parse(launchSettingsJsonContents))
+                var jsonDocumentOptions = new JsonDocumentOptions
```

Comment:
Could you add a test for a launchSettings.json with comments, so that we can cover this scenario and prevent this regressing again?

Reply:
@livarcocc I'm thinking we can just add a comment to the existing test asset, that should cover the scenario. I'll update the PR in the morning.

https://github.com/dotnet/cli/tree/95c6eff6daa1a69f29c42b2d405400ad44bdec91/TestAssets/TestProjects/AppWithLaunchSettings/Properties

Reply:
Sounds good to me.